scripts/testEmailFinder.sh

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+
+command=$HOME/furlough/src/find_email_addresses.py
+
+python3.5 ${command} www.jana.com/contact &> jana.txt 
```

Comment:
These tests are great. Were they helpful while developing?

Reply:
I was running ad-hoc tests during development. Those are not real software tests , such script comes handy when you want to run  "developed software"  overnight.